app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,14 @@
 class PostsController < ApplicationController
   # GET /users/posts
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @posts = @user.posts.includes(:comments)
+    @comments = Comment.new
+  end
```

Comment:
- [ ] Kindly not that this method should only be responsible for getting all post not for creating a new comment. if you wasn't to get comments on that post you can simple do @post.comments as these two table are associated in the database.